app/models/classified_listing.rb

```diff
@@ -67,7 +67,7 @@ def self.categories_available
       "education" => { cost: 1, name: "Education/Courses", rules: "Educational material and/or schools/bootcamps." },
       "jobs" => { cost: 25, name: "Job Listings", rules: "Companies offering employment right now." },
       "mentors" => { cost: 1, name: "Offering Mentorship", rules: "You are available to mentor someone." },
-      "products" => { cost: 5, name: "Products/Tools", rules: "Must be available right now." },
+      "products" => { cost: 10, name: "Products/Tools", rules: "Must be available right now." },
```

Comment:
In PR #2737 @benhalpern changed it to 5. Just trying to avoid any miscommunication 😉

Reply:
Yeah, we should probably get on the same page about that 🙃

Reply:
Oh i see you fixed the other location to match up.